docs/examples/device-flow.otg.yaml

```diff
@@ -0,0 +1,75 @@
+devices:
+- ethernets:
+  - ipv4_addresses:
+    - address: 192.0.2.1
+      gateway: 192.0.2.2
+      name: otg1.eth[0].ipv4[0]
+      prefix: 24
+    mac: 02:00:00:00:01:aa
+    mtu: 1500
+    name: otg1.eth[0]
```

Comment:
the `[]` notation is a bit confusing in the name.  Makes me think that we're referencing the 0th index in some list.  What about `eth0` instead of `eth[0]`?  Minor nit-pick.

Reply:
`eth0` would be confusing as it would imply you're using `eth0` interface somewhere (since such interface actually exists on ixia-c-te containers). By using `eth[0]` I'm making it clear this is an array of ethernets, and I'm referring to the first entry. So what you think is actually correct.